### PR TITLE
Update Documentation For New Septentrio-Specific Driver

### PR DESCRIPTION
**Pull request by flyingthingsintothings**
As the Septentrio driver [has moved from the PX4 GPS drivers repository into the main codebase](https://github.com/PX4/PX4-Autopilot/pull/22904) and received new parameters and features, the documentation needs to be updated to reflect them. I checked the changes with `yarn start` and everything looks normal. There is one missing link (see bot reply below) which is just a future link that will point to the new parameters once they are available in the documentation, as that will automatically happen from what I understand.

### CODE REVIEW

File: en/gps_compass/septentrio.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+::: warning
+By default, the `GPS` module will be configured to use the `GPS 1` port.
+Make sure to set `GPS_1_CONFIG` to `Disabled`.
+:::
+
+PX4 should then automatically configure the connected receiver(s), after which the position will become visible in _QGroundControl_ and its GPS icon will display general status information.
+
+Note that you can also use a secondary Septentrio GNSS module, which is configured in a similar way.
+It is common to connect a second module to the port labeled `GPS 2`
+The port is configured using the [SEP_PORT2_CFG](../advanced_config/parameter_reference.md#SEP_PORT2_CFG) parameter.
```
Comment (hamishwillee) [inline en/gps_compass/septentrio.md:67]: FYI, I added this.

**Comment by hamishwillee** [comment]
Thanks @flyingthingsintothings this is good thanks. I have subedited it for style - in particular for your note:
1. I break lines on sentences rather than line length. Makes translation easier. Also use prettier for formatting.
2. I link parameters - makes it easy to check removed components in future. IN general using linking is a "good thing".

**Review by hamishwillee — APPROVED** [review]
@flyingthingsintothings OK, I'm going to merge, but probably a good idea for you to look at the current version and make sure my edits didn't change anything "incorrectly".

Note, one thing missing, which you can do as a post process. I think you need to add EKF2 settings to enable yaw, such as these https://docs.px4.io/main/en/gps_compass/rtk_gps_holybro_unicore_um982.html#enable-gps-heading-yaw

**Comment by flyingthingsintothings** [comment]
@hamishwillee I didn't find in the contribution guidelines that Prettier was used for formatting. That could maybe be added. Thank you for the edits. I always use max line length but like you say, splitting on sentences does make more sense.

**Comment by hamishwillee** [comment]
@flyingthingsintothings I guess it is a bit unreasonable for you to guess the requirements :-). Thanks for the timely reminder!

I have made an update in https://github.com/PX4/PX4-user_guide/pull/3261

**Comment by julianoes** [comment]
Thanks for this @flyingthingsintothings, helped me today! :heart:

**Comment by julianoes** [comment]
@hamishwillee this PR needs backporting to v1.15! It looks like the feature was shoved into v1.15 last minute.

**Comment by hamishwillee** [comment]
@julianoes Well that's annoying. I manually removed it after the first cut of the build because it wasn't present :-(

**Pull request merged**